### ISSUE TITLE
Exit Nodes are enabled by default

### ISSUE THREAD

**Issue by W1BTR** [netbirdio/netbird#2155]
**Describe the problem**

If exit nodes are available, upon restarting the netbird client on PC, it automatically enables it.

**To Reproduce**

1. Sign into netbird on Windows
2. Check "Network Routes"

**Expected behavior**

Ideally, it saves whatever status it had on last run.

**Are you using NetBird Cloud?**

I self-host NetBird's control plane.

**NetBird version**

`0.27.10`

**Comment by W1BTR** [comment]
Seems like exit nodes are also applying by default to devices in an exit node group on linux. There is an exit node given to the "All" group. When installing netbird on a server, all the services went down as it was forced to use the exit node by default.

Ideally this is something we can enable and disable on the fly. I'd like a handful of exit nodes available to users, without being enabled by defaults

**Comment by pascal-fischer** [comment]
Hi @W1BTR,
the first version of exit nodes is fully managed from the admin dashboard. We already planned on making the client remember if a route was disabled by a user or not. With that we will evaluate if and how to maybe mark them already on management to not be assigned by default.

**Comment by W1BTR** [comment]
> Hi @W1BTR, the first version of exit nodes is fully managed from the admin dashboard. We already planned on making the client remember if a route was disabled by a user or not. With that we will evaluate if and how to maybe mark them already on management to not be assigned by default.

Makes sense. The client remembering is definitely a big thing, glad it's being taken care of. Having an option to override this on the dashboard does sound powerful, allowing the admin to move servers and devices between virtual networks! Super cool!

**Comment by W1BTR** [comment]
The amount of times I think my networking is broken or there's a lot of lag in the network at work before I realize netbird is just running is quite often 🤣

Although Im not sure why the connection drops every 20 seconds when using one of my exit nodes. I need to test if it always has this issue or just with this node.

**Comment by mlsmaycon** [comment]
Hey @W1BTR , can you leave your comments here: https://github.com/netbirdio/netbird/issues/2193

We will be updating the behavior and would be great to collect all in one place.

**Comment by nazarewk** [comment]
is it still relevant after introduction of Exit Node selection mechanism?

**Comment by anarh1st47** [comment]
This is still works the same

**Comment by 4piu** [comment]
The netbird client on my macOS automatically re-enabled all exit nodes for unknown reason, interrupting internet. I am not sure this is fully fixed.

**Comment by 4piu** [comment]
I encountered this problem again in Arch with `v0.71.4`. Whenever netbird is connected, it turns on all networks by default. Even after I deselect all networks using both cli and netbird-ui the settings cannot persists. There's only one file `.config/netbird/active_profile.txt`. 

I cannot believe there isn't a global kill switch that disable this annoying and frustrating stupid feature in netbird control panel. In my experience this problem occurs so many times on almost all platforms and causing so many damn connectivity trouble for me.

**Comment by BigTimeRadio** [comment]
As of a more recent version there is a setting for this, it is when you set up the exit node, you can unselect "Enabled by default."
